### PR TITLE
add support for rails5

### PR DESCRIPTION
**Pull request by lsylvester**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 end
 
 Rails.application.routes.draw do
-  unless Rails.application.routes.named_routes.routes[:status]
+  unless Rails.application.routes.named_routes[:status]
```
Comment (olivierlacan) [inline config/routes.rb]: Is this backward compatible with earlier versions of Rails?

Comment (nbibler) [inline config/routes.rb]: Seems to pass in Rails 3.x and 4.x. :+1:

**Comment by nbibler** [comment]
I'm wondering what the best way to support this in the short term is. We're using [Appraisal](https://rubygems.org/gems/appraisal) on this library, which works great. But, it seems weird to setup a test case for a beta/pre-release of a library.

Ideally, we'd setup and test for railties `~> 5.0`. However, that pessimistic lock does not include beta releases, so it doesn't work for the current `5.0.0.beta1` release.

**Comment by nbibler** [comment]
Further complicating the short term support: `rspec-rails` doesn't yet support Rails 5, either.

**Comment by nbibler** [comment]
Also, the `gemfiles/rails5.0.gemfile` file can be removed. It's auto-generated and replaced when Appraisal runs.

I see you were able to make it work by using GitHub checkouts of the rspec libraries. Seems bad. ;)

**Comment by nbibler** [comment]
For now, I've gone ahead and fixed the `.routes` issue on master (at db3ee0cf95cf907ab2c483baa6967075633270a5). I've also bumped the supported Rails version up to `< 6`. I don't yet have Travis/appraisal running Rails 5 tests... I'll try to remember to do that once it gets a stable release and rspec officially supports it.

**Comment by nbibler** [comment]
Thank you for the pull request and the heads up about the Rails 5 issue, @lsylvester !

**Comment by nbibler** [comment]
Version 3.4.0 was just released to include these changes.

**Comment by olivierlacan** [comment]
<3

**Pull request closed without merging**